core/src/main/java/org/hisp/dhis/android/core/D2.kt

```diff
@@ -78,6 +79,10 @@ class D2 internal constructor(internal val d2DIComponent: D2DIComponent) {
         return d2DIComponent.httpServiceClient
     }
 
+    fun httpServiceClientKotlinx(): HttpServiceClientKotlinx {
```

Comment:
Couldn't it be internal at least? It should avoid all the modifications in `HttpServiceClientKotlinx` done in this PR